tests/EvmAcceptanceTests/test/ContractInteraction.js

```diff
@@ -149,6 +149,17 @@ describe("Contract Interaction", function () {
         expect(await contract.methods.setAddress(ADDRESS).send(options)).to.be.not.null;
         expect(await contract.methods.someAddress().call()).to.be.eq(ADDRESS);
       });
+
+      it("Calling a method on a contract that generates event without parameters", async function () {
```

Comment:
please change `it` description based on our style. we already have `Should have the event argument in returned object when arg type is uint256` `Should have the event name in returned object` and ...

Reply:
@its-saeed Done.